linux/sound.cpp

```diff
@@ -195,14 +195,14 @@ void CSound::Stop()
 int CSound::Init ( const int /* iNewPrefMonoBufferSize */ )
 {
 
-    // clang-format off
-// try setting buffer size
-// TODO seems not to work! -> no audio after this operation!
-// Doesn't this give an infinite loop? The set buffer size function will call our
-// registered callback which calls "EmitReinitRequestSignal()". In that function
-// this CSound::Init() function is called...
-//jack_set_buffer_size ( pJackClient, iNewPrefMonoBufferSize );
-    // clang-format on
+    // TODO: BEGIN
+    /* try setting buffer size seems not to work! -> no audio after this operation!
+     * Doesn't this give an infinite loop? The set buffer size function will call our
+     * registered callback which calls "EmitReinitRequestSignal()". In that function
+     * this CSound::Init() function is called...
+     * jack_set_buffer_size ( pJackClient, iNewPrefMonoBufferSize );
+     */
```

Comment:
```suggestion
    /* try setting buffer size seems not to work! -> no audio after this operation!
     * Doesn't this give an infinite loop? The set buffer size function will call our
     * registered callback which calls "EmitReinitRequestSignal()". In that function
     * this CSound::Init() function is called...
     */
     // jack_set_buffer_size ( pJackClient, iNewPrefMonoBufferSize );
     
```

I think this would be clearer: Explanation what to do in "/*" and the commented code in "//" (at least if possible)